docs/user-guide/configuring-at-tls-for-zowe-server.md

```diff
@@ -1,4 +1,4 @@
-# Configuring AT-TLS for Zowe Server
+# Enabling AT-TLS
```

Comment:
If the title changes I think it should change the name of the file and the item in sidebars cc. @janan07

Reply:
I think changing filenames can break bookmarks and google indexing so in the past I have left filenames outdated just so users still end up on the right page.

Reply:
Could be, we already changed this one once, since the original title was API ML only. I think the search is not working properly in stable version atm

Reply:
Just a note: if we want to change the names of the files, we should move the file and keep only the link to the new file on the original page.